README.md

```diff
@@ -206,6 +213,9 @@ The shared storage worklet invocation methods (`addModule`, `createWorklet`, and
             *   The approximate size of the contents of this interest group, in bytes.
     *   The [AuctionAdInterestGroup](https://wicg.github.io/turtledove/#dictdef-auctionadinterestgroup)'s [lifetimeMs](https://wicg.github.io/turtledove/#dom-auctionadinterestgroup-lifetimems) field will remain unset. It's no longer applicable at query time and is replaced with attributes `timeSinceGroupJoinedMs` and `lifetimeRemainingMs`.
     *   This API provides the Protected Audience buyer with a better picture of what's happening with their users, allowing for Private Aggregation reports.
+*   `navigator.locks.request(resource, callback)` and `navigator.locks.request(resource, options, callback)`
+    *   Acquires a lock on `resource` and invokes `callback` with the lock held. See the [request](https://w3c.github.io/web-locks/#dom-lockmanager-request) method in Web Locks API for details.
+    *   Lock Scope: shared storage locks are partitioned by the shared storage data origin, and are independent of any locks obtained via `navigator.locks.request` in a Window or Worker context.
```

Comment:
Please mention that this is for privacy reasons. It should likely also get a sentence or two in the privacy section.

Reply:
Done.